poethepoet/plugin.py

```diff
@@ -0,0 +1,193 @@
+# pylint: disable=import-error
```

Comment:
I think we can remove this? This was to silence Pylint from the `try`/`except` hack I did before

Reply:
I added this to stop pylint complaining about cleo not being a direct dependency :)